libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp

```diff
@@ -909,6 +922,10 @@ bool AP_InertialSensor_Invensensev3::hardware_init(void)
         // disable STC
         uint8_t reg = register_read_bank_icm456xy(INV3BANK_456_IPREG_TOP1_ADDR, 0x68);  // I3C_STC_MODE b2
         register_write_bank_icm456xy(INV3BANK_456_IPREG_TOP1_ADDR, 0x68, reg & ~0x04);
+    } else if (inv3_type == Invensensev3_Type::ICM42688) {
+        // fix for the "stuck gyro" issue
+        const uint8_t v = register_read(INV3REG_42688_INTF_CONFIG1);
+        register_write(INV3REG_42688_INTF_CONFIG1, (v & 0x3F) | 0x40);
```

Comment:
should we make it a checked register?

Reply:
Given that Vishal indicated that this applies to all ICM42 and all IIM42 parts I think we should consider at least applying this to the 42605 which is used in a bunch of places.